groundstation/static/js/components/Countdown.jsx

```diff
@@ -153,6 +153,16 @@ class Countdown extends Component{
                     nextPassover: data.nextPassover
                 });
             });
+
+            // temporary place to send slack messages for passover
+      
+            fetch('/api/passovers', {
+              method: 'HEAD',
+              headers: {
+                'Content-Type': 'application/json',
+                'Authorization': 'Bearer '+ localStorage.getItem('auth_token')
+              }
+            });
```

Comment:
Can remove this if we are going to be calling the endpoint from the python automation module.

```suggestion

```